source/release-notes/1.0.txt

```diff
@@ -12,6 +12,20 @@ Release Notes for mongosync 1.0
    :depth: 1
    :class: singlecol
 
+.. warning:: Past Release Limitations
+
+   .. include:: /includes/critical-warning-text
+
+   .. list-table::
+      :header-rows: 1
+      :widths: 70 30
+
+      * - Issue
+        - Affected Versions
+
+      * - .. include:: /includes/release-notes/REP-4274
+        - 1.1.0 - 1.7.1
```

Comment:
Looks like you accidentally put this in the 1.0.txt file instead of the 1.1.txt file.

Reply:
Fixed